### PR TITLE
Update to nginx reverse proxy for ws

### PR DESCRIPTION
**Pull request by kurdin**
adding more reverse proxy settings to nginx
                proxy_set_header X-Forwarded-Ssl on;
                proxy_set_header Host $host

### CODE REVIEW

**Review by kurdin — COMMENTED** [review]
@svandeneertwegh @ruslandoga Please review PR.

File: reverse-proxy/nginx/plausible [hunk 1]
```diff
@@ -7,6 +7,8 @@ server {
 
 	location / {
 		proxy_pass http://127.0.0.1:8000;
+		proxy_set_header Host $host;
+                proxy_set_header X-Forwarded-Ssl on;
```
Comment (ruslandoga) [inline reverse-proxy/nginx/plausible:11]: I don't think `X-Forwarded-Ssl` would be used in Plausible in any way.

Comment (svandeneertwegh) [reply reverse-proxy/nginx/plausible:11]: I use:

proxy_http_version 1.1;
proxy_set_header Upgrade $http_upgrade;
proxy_set_header Connection "Upgrade";

Comment (ruslandoga) [reply reverse-proxy/nginx/plausible:11]: Right, that was added in https://github.com/plausible/community-edition/pull/113

Comment (kurdin) [reply reverse-proxy/nginx/plausible:11]: @ruslandoga  maybe you right and we don't need X-Forwarded-Ssl. Lets remove it

Comment (kurdin) [reply reverse-proxy/nginx/plausible:11]: PS. Notice that PR is closed `X-Forwarded-Ssl` would not do any bad, even if it works without it.

**Pull request closed without merging**